portal-api/src/v1/api.js

```diff
@@ -800,6 +820,7 @@ const putPortalFacilityAmendment = async ({ dealId, facilityId, amendment, audit
  * @param {string} params.checkersEmail - The checker's email address to send the notification to
  * @param {string} params.pimEmail - The pim's email address to send the notification to
  * @param {import('@ukef/dtfs2-common').PortalAmendmentSubmittedToUkefEmailVariables} params.emailVariables - The email variables to send with the notification
+ * @param {number} params.requestDate - The date of the request
```

Comment:
```suggestion
 * @param {number} params.requestDate - The date the bank requested the amendment - the date it was submitted to UKEF.
```
can you add the same you had and in the other function

Reply:
done